packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -416,7 +419,22 @@ module.exports = function (log, config) {
     );
   };
 
-  Mailer.prototype.localize = function (message) {
+  Mailer.prototype.localize = async function (message) {
+    if (isNewEmails === true) {
+      const { localizedHTML, localizedText, localizedSubject } =
+        await this.fluentLocalizer.localizeEmail(
+          message.template,
+          'hello-world'
```

Comment:
@jodyheavener @LZoog Temporarily, I'm passing this string as a subject since I wasn't sure where exactly it's being created